src/lib/containers/entity_finder/tree/EntityTree.tsx

```diff
@@ -50,12 +57,22 @@ function getScopeOptionNodeName(scope: FinderScope): string {
       return 'Favorites'
   }
 }
+
+/**
+ * Indicates which container is selected when the tree is used to control another component. If 'root', then the top-level entities
+ * should be shown in the other component. If a synID, then that entity's children should be shown in the other component. If null,
+ * then a selection has not been made.
+ */
+export type EntityTreeContainer = string | 'root' | null
+
 // if the first item is selected (matching the dropdown), then output a configuration. otherwise, output a synId
 export type EntityTreeProps = {
   initialScope?: FinderScope
   /** To show the current project, projectId must be defined */
   projectId?: string
-  initialContainer: string | 'root' | null
+  initialContainer: EntityTreeContainer
```

Comment:
It may seem odd that `initialContainer` is still passed, but some logic still exists at this level that checks the initial container. The logic hasn't been moved because it explicitly deals with populating the tree.